src/main/java/soolutions/hometaskapp/core/Amount.java

```diff
@@ -0,0 +1,40 @@
+package soolutions.hometaskapp.core;
+
+// TODO(vivek): Write docs for each public class
```

Comment:
Why have docs? Shouldn't the tests be the documentation?

Reply:
It depends, sometimes people just want an overview of what a class does.

Reply:
That makes sense.